### PR TITLE
(feat) translated api platform article

### PR DESCRIPTION
**Pull request by GusTTShowbiz**

### CODE REVIEW

File: _posts/2017-08-24-api-platform-en.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+To generate our entity, we only need to execute the following command:
+
+```
+vendor/bin/schema generate-types src/ app/config/schema.yml
+```
+
+Yay ! If you go to src/AppBundle/Entity/Movie.php, we get an automatic generated entity that has all kinds of validation according to the types set in _Schema.org_.
+
+Last be not least, we need to update the database to be able to play with our movie API:
```
Comment (ch3ric) [inline _posts/2017-08-24-api-platform-en.md]: Last but

File: _posts/2017-08-24-api-platform-en.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+```yaml
+services:
+    movies.search_filter:
+        parent:    'api_platform.doctrine.orm.search_filter'
+        arguments: [ { id: 'exact', name: 'partial', countryOfOrigin: 'partial' } ]
+        tags:      [ { name: 'api_platform.filter', id: 'movies.search_filter' } ]
+```
+
+As you can see, we define for all properties the level of comparison: for the _id_, we want an _exact_ match (id=5), for the name a _partial_ level, which is a LIKE %name%, and the same thing for our _countryOfOrigin_. Other levels are available to strings, but not only, you have numeric values, booleans, dates...
```
Comment (ch3ric) [inline _posts/2017-08-24-api-platform-en.md]: Available to strings?

File: _posts/2017-08-24-api-platform-en.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+Api Platform keeps its promise concerning how fast we can develop an API: creating an API in a few minutes is a child's play!
+
+Developed using best practices, API Platform is easily extensible, which allows everyone to add behaviour.
+
+Finally, the documentation is a very nice thing. You have plenty of examples and explanations.
+
+## To come
+
+Version 2.1 of API platform will be soon released, and comes with new features such as: an admin system developed in React, new filters... Details [here](https://dunglas.fr/2017/06/api-platform-2-1-feature-walkthrough-create-blazing-fast-hypermedia-apis-generate-js-apps/)
```
Comment (ch3ric) [inline _posts/2017-08-24-api-platform-en.md]: Might be released very soon, we have to publish this post asap ;)

File: _posts/2017-08-24-api-platform-en.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+      countryOfOrigin: { range: "Text" }
+```
+
+To generate our entity, we only need to execute the following command:
+
+```
+vendor/bin/schema generate-types src/ app/config/schema.yml
+```
+
+Yay ! If you go to src/AppBundle/Entity/Movie.php, we get an automatic generated entity that has all kinds of validation according to the types set in _Schema.org_.
```
Comment (eko) [inline _posts/2017-08-24-api-platform-en.md]: I think you should add some ` `` ` chars for `src/AppBundle/Entity/Movie.php`

Comment (eko) [inline _posts/2017-08-24-api-platform-en.md]: `we get an automatic generated entity` -> `we get an entity generated automatically`

File: _posts/2017-08-24-api-platform-en.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+```
+You can now query your API with the following URL: `movies?name=O&order[name]=desc`  
+Very simple to implement right?
+
+Obviously, you can create your own [Api Platform filters](https://api-platform.com/docs/core/filters#creating-custom-filters).
+
+Many other features exist like :
+* [pagination](https://api-platform.com/docs/core/pagination)
+* a rich event system
+* an cache invalidation system (coming in version 2.1)
```
Comment (eko) [inline _posts/2017-08-24-api-platform-en.md]: `an cache` -> `a cache`

File: _posts/2017-08-24-api-platform-en.md (hunk 3 above)
Comment (eko) [inline _posts/2017-08-24-api-platform-en.md]: `will be soon released` -> `will soon be released`? Not sure but it sounds better

Comment (eko) [inline _posts/2017-08-24-api-platform-en.md]: Yep, Kevin Dunglas will talk about the React frontend app at Symfony Live London on September 21th

Comment (GusTTShowbiz) [inline _posts/2017-08-24-api-platform-en.md]: yes it'll be published thursday :)

File: _posts/2017-08-24-api-platform-en.md (hunk 1 above)
Comment (GusTTShowbiz) [inline _posts/2017-08-24-api-platform-en.md]: haha j'ai pas branché mon cerveau

File: _posts/2017-08-24-api-platform-en.md (hunk 3 above)
Comment (GusTTShowbiz) [inline _posts/2017-08-24-api-platform-en.md]: yes you're right i went to fast

**Comment by GusTTShowbiz** [comment]
@ch3ric @eko Feedbacks applied ! sorry for the silly mistakes

**Comment by ch3ric** [comment]
@ElevenTom note: this post has to be published before Thursday. otherwise the conclusion will be out-dated :)

**Comment by ElevenWilson** [comment]
13 words might be misspelled, please check them: api, API, APIs, AppBundle, bool, booleans, countryOfOrigin, desc, png, rpierlot, src, symfony, Yay

**Comment by ElevenWilson** [comment]
13 words might be misspelled, please check them: api, API, APIs, AppBundle, bool, booleans, countryOfOrigin, desc, png, rpierlot, src, symfony, Yay

**Comment by ElevenWilson** [comment]
13 words might be misspelled, please check them: api, API, APIs, AppBundle, bool, booleans, countryOfOrigin, desc, png, rpierlot, src, symfony, Yay

**Pull request merged**